--- src/common/servlist.c	2011-04-11 11:19:50.000000000 +0200
+++ src/common/servlist.c.oden	2011-04-11 11:23:15.771624405 +0200
@@ -213,7 +213,16 @@ static const struct defaultserver def[]
 	{0,			"irc.vendetta.com"},
 
 	{"FreeNode",	0, "#mandriva"},
-	{0,				"irc.freenode.net"},
+	{0,				"irc.freenode.net/6665"},
+	{0,				"irc.freenode.net/6666"},
+	{0,				"irc.freenode.net/6667"},
+	{0,				"irc.freenode.net/8000"},
+	{0,				"irc.freenode.net/8001"},
+	{0,				"irc.freenode.net/8002"},
+#ifdef USE_OPENSSL
+	{0,				"irc.freenode.net/+7000"},
+	{0,				"irc.freenode.net/+7070"},
+#endif
 
 /*	{"Freeworld",	0},
 	{0,			"kabel.freeworld.nu"},
